runtime-config-linux.md

```diff
@@ -130,6 +130,13 @@ Next parameters can be specified:
 Also known as cgroups, they are used to restrict resource usage for a container and handle
 device access.  cgroups provide controls to restrict cpu, memory, IO, and network for
 the container. For more information, see the [kernel cgroups documentation](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt).
+The path to the cgroups can be specified in the Spec. The path is expected to be relative
+to the cgroups mount point. The Spec does not support split hierarchy. The cgroups will be 
+created if they don't exist. 
```

Comment:
This should be [one line per sentence][1].  I'll file a PR suggesting we move stuff like that from the README to a `CONTRIBUTING.md`

[1]: https://github.com/opencontainers/specs/tree/138deee1418d2376a4e9d4966e9f1543e717a641#markdown-style